cypress/component/category-toolgroup.cy.js

```diff
@@ -0,0 +1,234 @@
+
+            commit: () => {},
+            state: state,
+            getters: getters
+        };
+    });
+
+    it("if nothing is selected or mapped, component is empty", () => {
+        cy.mount(categoryToolGroup, {
+            mocks: { $store: store },
+            computed: {
+                getColumnsForCategory: () => (p_category) => {
+                    return [];
+                },
+                getSelectedTools: () => {
+                    return [];
+                }
+            }
+        });
+
+        cy.get("[data-cy='toolgroup-select']").should('not.exist');
+        cy.get("[data-cy='assessment-tool-table']").should('not.exist');
+        cy.get("[data-cy='assessment-column-table']").should('not.exist');
+
+    });
+
+    it("gets columns about assessment tools from the store and shows them to me", () => {
+
+        cy.mount(categoryToolGroup, {
+            mocks: { $store: store },
+            computed: Object.assign(store.getters, {getSelectedTools: () => {
+                return [];
+            }})
+        });
+
+        cy.get("[data-cy='assessment-column-table']").should("be.visible");
+        cy.get("[data-cy='assessment-column-table']").contains("column1");
+        cy.get("[data-cy='assessment-column-table']").contains("column3");
+    });
+
+    it("before a tool is selected, all columns are unstyled", () => {
+        store.state.columnToToolMap = {
+                column1: null,
+                column2: null,
+                column3: null
+        };
+
+        cy.mount(categoryToolGroup, {
+            mocks: { $store: store },
+            computed: Object.assign(store.getters, {getSelectedTools: () => {
+                return [];
+            }})
+        });
+
+        cy.get("[data-cy='assessment-column-table']").find("tr:contains('column1')").should('not.have.class', 'selected-tool');
+        cy.get("[data-cy='assessment-column-table']").find("tr:contains('column3')").should('not.have.class', 'selected-tool');
+
+    });
+
+    it("gets assessment tool names from the store and shows them in a dropdown", () => {
+        cy.mount(categoryToolGroup, {
+            mocks: { $store: store },
+            computed: store.getters
+        });
+
+        cy.get("[data-cy='toolgroup-select']").should("be.visible");
+        cy.get("[data-cy='toolgroup-select']").type("MOCA{enter}");
+        cy.get("[data-cy='toolgroup-select']").should("contain", "MOCA");
+    });
+
+    it("selecting a tool group fires a createTool mutation", () => {
+        cy.spy(store, "commit").as("commitSpy");
+        cy.mount(categoryToolGroup, {
+            mocks: { $store: store },
+            computed: Object.assign(store.getters, {getSelectedTools: () => {
+                return [];
+            }})
+        });
+
+        cy.get("[data-cy='toolgroup-select']").type("MOCA{enter}");
+        cy.get("@commitSpy").should("have.been.calledWith", "createToolGroup", { identifier: 'cogatlas:MOCA', label: 'MOCA' });
```

Comment:
Forgot to update the test after renaming the mutation.

Reply:
ah, good catch. thanks. I used the smart refactor, guess it missed this